targeted/search cvedetails using target server header.js

```diff
@@ -0,0 +1,13 @@
+// Captures Server header from the application response and searches cvedetails.com for known target server vulnerabilities.
+
+
+function invokeWith(msg) {
+	host = msg.getRequestHeader().getURI().getHost(); 
+if (msg) 
+	{
+		headers = msg.getResponseHeader().getHeaders("Server")
+		org.zaproxy.zap.utils.DesktopUtils.openUrlInBrowser(
+		"http://www.cvedetails.com/google-search-results.php?q=" + headers + "&sa=Search");
```

Comment:
The value of ```headers``` should be URL encoded otherwise it might fail to create a valid URL (or the one intended).